### PR TITLE
Fix sample Redis

### PR DESCRIPTION
**Pull request by ilyar**
ReferenceError: redis is not defined

### CODE REVIEW

**Comment by crandmck** [comment]
This seems like the exact same thing.  Why would it work and not the original?

**Comment by ilyar** [comment]
@crandmck  ReferenceError: redis is not defined in

**Comment by caiobrentano** [comment]
Hi @ilyar. I think you tagged the wrong person! :)

File: en/guide/database-integration.md [hunk 1]
```diff
@@ -319,7 +319,8 @@ $ npm install redis
 ### Example
 
 ```js
-var client = require('redis').createClient()
+var redis = require("redis")
```
Comment (dougwilson) [inline en/guide/database-integration.md]: Use single quotes here for https://standardjs.com/

**Review by dougwilson — CHANGES_REQUESTED** [review]
Yea, this is correct, since below the direct change, there is use of the variable `redis` for `redis.print`, so the whole example seems to expect a `redis` variable.

**Review by ilyar — COMMENTED** [review]
up

**Comment by crandmck** [comment]
Oh, I see.    Sorry I didn't look at the rest of the code sample.. sorry.

OK LGTM, thanks!  I'll land it since you addressed Doug's review comment.

**Pull request merged**